src/modules/launcher/Wox.Plugin/ThemeManager.cs

```diff
@@ -178,7 +183,7 @@ public void Dispose()
 
     public enum Theme
     {
-        None,
+        System,
```

Comment:
It would be better to make `System` a seperate entity. In change theme function `currentTheme ` is set to `System`, however it is immediately changed. Ideally, theme should have one to one mapping with system themes. I would suggest to extract 
```
if (WindowsThemeHelper.IsHighContrastEnabled())
                {
                    Theme highContrastBaseType = GetHighContrastBaseType();
                    ChangeTheme(highContrastBaseType);
                }
                else
                {
                    string baseColor = WindowsThemeHelper.GetWindowsBaseColor();
                    ChangeTheme((Theme)Enum.Parse(typeof(Theme), baseColor));
                }
```
into separate function and call this from `SettingsWatcher`, if system default option is selected.